### PR TITLE
Getting Started with Synthetic Monitoring > Browser Tests

### PR DESCRIPTION
**Pull request by alai97**
How do we feel about the added section about sample results?

### What does this PR do?

Revamp the Getting Started with Browser Tests page with updated content.

### Motivation

DOCS-1979

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-browser-tests-getting-started/getting_started/synthetics/browser_test

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/getting_started/synthetics/browser_test.md [hunk 1]
```diff
@@ -19,77 +22,115 @@ further_reading:
...
-1. Navigate to one of the furniture sections, for instance **Chairs**, and select **Add to cart**.
-2. Click on **Cart**, click **Checkout**.
-3. Manually add the **Assertion** “Test text is present on the active page” to confirm the words “Thank you” are on the page.
+The example below demonstrates how to map a user journey from adding an item to a cart to successfully checking out:
 
-   **Note**: Your final browser test step must be an **assertion**. This will ensure your test ended up on an expected page and found the expected element.
+1. Navigate to one of the furniture sections on the example website such as **Chairs** and select **Add to cart**.
+2. Click on **Cart** and **Checkout**.
+3. Under **Add New**, select **Assertion** and click **“Test that some text is present on the active page”**.
+4. To confirm the words “Thank you!” appear after checking out, enter "Thank you!" in the **Value** field.
```
Comment (alai97) [inline content/en/getting_started/synthetics/browser_test.md:100]:
```suggestion
4. To confirm the words “Thank you!” appear after checking out, enter `Thank you!` in the **Value** field.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 2]
```diff
@@ -19,77 +22,115 @@ further_reading:
...
 
 ## Test results
 
-The browser test details page includes details about your test configuration, test uptime, historical graphs for response time and time to interactive for the first page, sample successful and failed results, and a list of test results corresponding to the selected timeframe. Each individual test result includes screenshots, core web vitals, potential errors, resources, and traces for each step.
+You can wait for your test to generate results in **Test Results** or click **Run test now** on the top right to trigger results quickly.
+
+To troubleshoot a failed test, scroll down to **Test Results** and click on a failed test result. Review screenshots, core web vitals, potential errors, resources, and traces for each step to diagnose the issue. 
+
+When you look at screenshots leading up to the failed step, you can identify the root cause of the failure more easily. You can also click on resources, errors, and traces that appear next to the duration graph to access **Errors & Warnings**, **Resources**, and **Traces**.
```
Comment (alai97) [inline content/en/getting_started/synthetics/browser_test.md:121]:
```suggestion
When you look at screenshots leading up to the failed step, you can identify the root cause of the failure more easily. Click on resources, errors, and traces that appear next to the duration graph to access **Errors & Warnings**, **Resources**, and **Traces**. 
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 3]
```diff
@@ -19,77 +22,115 @@ further_reading:
...
+In the example below, the test failed as the result of a server timeout.
 
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run with a trace generated from the test run. 
+
+To link browser test results with APM, whitelist the URLs you want to add to the APM integration headers. For wildcards, use `*`. For example, `https://*.datadoghq.com/*`
```
Comment (alai97) [inline content/en/getting_started/synthetics/browser_test.md:133]:
```suggestion
To link browser test results with APM, whitelist the URLs you want to add to the APM integration headers. 

For wildcards, use `*`. For example, `https://*.datadoghq.com/*`
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 4]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run with a trace generated from the test run. 
+
+To link browser test results with APM, whitelist the URLs you want to add to the APM integration headers. 
+
+For wildcards, use `*`. For example, `https://*.datadoghq.com/*`
```
Comment (alai97) [inline content/en/getting_started/synthetics/browser_test.md:135]:
```suggestion
For wildcards, use `*`. For example, `https://*.datadoghq.com/*`.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 5]
```diff
@@ -19,77 +22,117 @@ further_reading:
 
 ## Create a browser test
 
-[Browser tests][1] are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These tests **verify both that your applications are up and responding to requests**, and that key business transactions can be performed by users as expected.
+[Browser tests][1] are scenarios that Datadog executes on your web applications. You can configure periodic intervals to run tests from multiple locations and devices. These tests verify that your applications are responding to requests and users can perform business transactions as expected.
 
-In this example, a browser test is configured to map a user's journey from adding an item to cart to successful checkout. Each test execution is recorded in Datadog as a **Test Result**.
+The example below demonstrates the creation of a browser test, which maps a user's journey from adding an item to a cart to successfully checking out.
```
Comment (kayayarai) [inline content/en/getting_started/synthetics/browser_test.md:27]:
```suggestion
The example below demonstrates the creation of a browser test that maps a user's journey from adding an item to a cart to successfully checking out. 
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 6]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
-## Record your test steps
+### Create recording
+
+Once your test configuration has saved, Datadog prompts you to download and install the [Datadog test recorder][5] extension. You can run browser tests on Google Chrome, Microsoft Edge, and Firefox. 
 
-Once the test configuration is saved, Datadog will prompt you to download the [Datadog test recorder][5] extension. Browser tests can be only recorded on **[Google Chrome][6]**. Download and install the extension.
+To begin recording your test steps, click **Start Recording**. 
 
-Once this extension is installed, begin recording your test steps by clicking the **Start Recording** button. Navigate your page in the iframe to the right of the recording options. When you select a div, image, or any area of your page, the steps are recorded and used to create steps within the browser test. Learn more about each step in the [browser test steps doc][7].
+You can navigate through your page in the iframe to the right of the recording button. By selecting a div, image, or any area of your page, Datadog records and creates your steps in the browser test. For more information, see [Browser Test Steps][7].
```
Comment (kayayarai) [inline content/en/getting_started/synthetics/browser_test.md:91]:
```suggestion
Navigate through the page in the iframe to the right of the recording button. When you select a div, image, or any area of the page, Datadog records and creates your steps in the browser test. For more information, see [Browser Test Steps][7].
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 7]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
 
-For example, to record test steps that map a user's journey from adding an item to cart to successful checkout:
+To end recording your test steps, click **Stop Recording**.
 
-1. Navigate to one of the furniture sections, for instance **Chairs**, and select **Add to cart**.
-2. Click on **Cart**, click **Checkout**.
-3. Manually add the **Assertion** “Test text is present on the active page” to confirm the words “Thank you” are on the page.
+The example below demonstrates how to map a user journey from adding an item to a cart to successfully checking out:
```
Comment (kayayarai) [inline content/en/getting_started/synthetics/browser_test.md:95]: Might be helpful to provide the link to the Shopist website here? Is that the example website we're talking about?

File: content/en/getting_started/synthetics/browser_test.md [hunk 8]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+In the example below, the test failed as the result of a server timeout.
 
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run with a trace generated from the test run. 
+
+To link browser test results with APM, whitelist the URLs you want to add to the APM integration headers.
```
Comment (kayayarai) [inline content/en/getting_started/synthetics/browser_test.md:133]:
```suggestion
To link browser test results with APM, add the allowlist the URLs you want to add to the APM integration headers. 
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 9]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+**Errors and Warnings** display JavaScript and network errors, **Resources** display the resource providing this status, and **Traces** map the entirety of the request in seconds. 
+
+In the example below, the test failed as the result of a server timeout.
 
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run with a trace generated from the test run.
```
Comment (kayayarai) [inline content/en/getting_started/synthetics/browser_test.md:131]: I think you removed or changed some key phrases here that were better before, specifically about the Traces tab, and "it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run."

**Review by kayayarai — CHANGES_REQUESTED** [review]
Lots of good changes! A few edit comments.

File: content/en/getting_started/synthetics/browser_test.md (hunk 7 above)
Comment (alai97) [reply content/en/getting_started/synthetics/browser_test.md:95]:
```suggestion
The example below demonstrates how to map a user journey from adding an item to a cart to successfully checking out in `https://www.shopist.io`:
```

File: content/en/getting_started/synthetics/browser_test.md (hunk 9 above)
Comment (alai97) [inline content/en/getting_started/synthetics/browser_test.md:131]:
```suggestion
With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab. 
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 10]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+### Configure your test details
 
 1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **Synthetic Test**.
 2. In the top right corner, click the **New Test** button.
 3. Select **Browser Test**.
-4. Define the configuration of your browser test:
+4. Define your browser test:
+
+    - Add the URL of the website you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io`, a test web application.
+    - You can select **Advanced Options** to use custom request headers, authentication credentials, proxy URLs, or cookies.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:41]:
```suggestion
    - Select **Advanced Options** to use custom request headers, authentication credentials, proxy URLs, or cookies. 
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 11]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
 
 1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **Synthetic Test**.
 2. In the top right corner, click the **New Test** button.
 3. Select **Browser Test**.
-4. Define the configuration of your browser test:
+4. Define your browser test:
+
+    - Add the URL of the website you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io`, a test web application.
+    - You can select **Advanced Options** to use custom request headers, authentication credentials, proxy URLs, or cookies. 
+    - You can set environment tags such as `env:prod` and `app:shopist` and additional tags on your test. Tags allow you to stay organized and quickly find tests you're interested in on the homepage.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:42]:
```suggestion
    - Set environment tags, for example `env:prod` and `app:shopist`, and additional tags for your test. Tags allow you to stay organized and quickly find tests you're interested in on the homepage.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 12]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+
+Select one or more **Managed Locations** or **Private Locations** to run your test from.
+
+Your managed locations often include public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select **Private Locations** instead.
+
+For more information, see [Getting Started with Private Locations][10].
+
+#### Specify test frequency
+
+Select the frequency of your test runs.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:55]:
```suggestion
Select the frequency that your test runs.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 13]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+Select the circumstances in which you want a test to send notification alerts.
 
-    - **Starting URL**: Add the URL of the website you’re interested in monitoring. If you don’t know what to start with, you can use `https://www.shopist.io` as a test web application.
-    - **Name**: Name the test.
-    - **Tags**: You can set tags such as `env:prod` and `app:shopist` on your test. Tags help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
-    - **Browsers & Devices**: Choose the devices and browsers you want to use for testing. In this example, the test is only run from **Chrome** and on **Large Laptops**.
-    - **Locations**: Choose one of the **Managed Locations** to run your test from. In this example, the test is run in **Americas and Europe**.
-    - **Specify a test frequency**: Select how often you would like the test to run.
-    - **Alert Conditions**: Set alert conditions to determine the circumstances under which you want a test to send a notification alert.
+You can specify the number of failures per browser and device before the test is considered failed:
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:61]:
```suggestion
Specify the number of failures per browser and device before the test is considered failed:
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 14]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+```
 
-         **Note**: By default, there is a 300ms wait before retrying a test that failed. This interval can be configured via the [API][3].
+#### Notify your team
 
-    - **Notify**: Write an alert message and specify which email addresses should be notified when the alert is triggered. No additional set up is required to start receiving alert emails from Datadog. You can also use [integrations][4], such as Slack, PagerDuty, webhooks, etc., to receive alert notifications.
-    - Click **Save & Edit Recording**.
+Add an alert name to the **Monitor Name** field and write a message for the alert. You can tag other teams, specify which service and team members receive the alert notifications, and use [integrations][4] such as Slack, PagerDuty, and webhooks.
+
+You can set time for your alert notification to re-notify if the alert has not been resolved and define the priority of the alert, ranging from **P5 (Info)** to **P1 (Critical)**.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:79]:
```suggestion
Set a time for your alert notification to re-notify if the alert has not been resolved, and define the priority of the alert, ranging from **P5 (Info)** to **P1 (Critical)**.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 15]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+
+When you're ready to run your test, click **Save Details & Record Test**.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:81]:
```suggestion
When you're ready to record your test, click **Save Details & Record Test**.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 16]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
 
 {{< img src="getting_started/synthetics/configured-browser-test.gif" alt="Configured browser test"  style="width:90%;">}}
 
-## Record your test steps
+### Create recording
+
+Once your test configuration has saved, Datadog prompts you to download and install the [Datadog test recorder][5] extension. You can run browser tests on Google Chrome, Microsoft Edge, and Firefox.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:87]:
```suggestion
Once your test configuration has saved, Datadog prompts you to download and install the [Datadog test recorder][5] extension. Browser tests are able to run on Google Chrome, Microsoft Edge, and Firefox. 
```

Comment (sarina-dd) [reply content/en/getting_started/synthetics/browser_test.md:87]: Do we have links to Microsoft Edge / Firefox? This link is for the Chrome extension.

File: content/en/getting_started/synthetics/browser_test.md [hunk 17]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+To troubleshoot a failed test, scroll down to **Test Results** and click on a failed test result. Review screenshots, core web vitals, potential errors, resources, and traces for each step to diagnose the issue. 
 
-The **Errors & Warnings** tab provides a list of Javascript and network errors, the **Resources** tab locates the resource providing this status, and the **Traces** tab maps the entirety of the request in seconds. This test failed as the result of a server timeout. The resource, `https://api.shopist.io/checkout.json`, posted the status ,and the targeted source of the problem is a controller linked to checkout. You have now successfully found the route of the problem.
+When you look at screenshots leading up to the failed step, you can identify the root cause of the failure more easily. Click on resources, errors, and traces that appear next to the duration graph to access **Errors & Warnings**, **Resources**, and **Traces**.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:121]:
```suggestion
 To identify the root cause of the failure more easily, look at screenshots leading up to the failed step. Click on resources, errors, and traces that appear next to the duration graph to access **Errors & Warnings**, **Resources**, and **Traces**. 
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 18]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+**Errors and Warnings** display JavaScript and network errors, **Resources** display the resource providing this status, and **Traces** map the entirety of the request in seconds. 
+
+In the example below, the test failed as the result of a server timeout.
 
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab.
```
Comment (sarina-dd) [inline content/en/getting_started/synthetics/browser_test.md:131]:
```suggestion
With Datadog's [APM integration with Synthetic Monitoring][8],  access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab. 
```

**Review by sarina-dd — COMMENTED** [review]
Editorial review completed 🍾

File: content/en/getting_started/synthetics/browser_test.md (hunk 16 above)
Comment (alai97) [reply content/en/getting_started/synthetics/browser_test.md:87]: And I oop! Good catch on the link, I'll change it to Google Chrome (https://www.google.com/chrome/).

@sarina-dd is there a way to remove the EN-US part of these page URLs?

https://www.mozilla.org/en-US/firefox/new/
https://www.microsoft.com/en-us/edge

**Review by sarina-dd — APPROVED** [review]

File: content/en/getting_started/synthetics/browser_test.md [hunk 19]
```diff
@@ -19,77 +22,117 @@ further_reading:
...
+**Errors and Warnings** display JavaScript and network errors, **Resources** display the resource providing this status, and **Traces** map the entirety of the request in seconds. 
+
+In the example below, the test failed as the result of a server timeout.
 
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8],  access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab.
```
Comment (alai97) [inline content/en/getting_started/synthetics/browser_test.md:131]:
```suggestion
With Datadog's [APM integration with Synthetic Monitoring][8], access the root cause of a failed test run by looking at the trace generated from the test run in the **Traces** tab. 
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**